### ISSUE TITLE
Bug at sqlserver2pgsql.pl line 2085 when creating views

### ISSUE THREAD

**Issue by McDoit** [dalibo/sqlserver2pgsql#55]
After stepping by the issue with sparse columns i ran into this issue:

```
Line <create view
> (4545) not understood. This is a bug at sqlserver2pgsql.pl line 2085, <$file> line 4545.
```

Which creates a CTE view with all ancestors to a place

The code for the view looks like this:

```
GO
/****** Object:  View [core].[PlaceAncestors]    Script Date: 2017-02-21 09:50:40 ******/
SET ANSI_NULLS ON
GO
SET QUOTED_IDENTIFIER ON
GO
create view 
[core].[PlaceAncestors]
as 

with parents as 
(
  select Id BaseId, Id, ParentId, Name, TypeId
  from core.Places cp2
  where ParentId is not null
  union all 
  select p.BaseId, cp1.Id, cp1.ParentId, cp1.Name, cp1.TypeId
  from parents p
    inner join core.Places cp1 on p.ParentId = cp1.Id
      and (cp1.Id <> cp1.ParentId or cp1.ParentId is null)	    
)
select IsNull(BaseId,0) BaseId, IsNull(Id,0) Id, ParentId, IsNull(Name,'') Name, ISNULL(TypeId, 100) TypeId
from parents
where BaseId <> Id


GO
/****** Object:  View [dbo].[vwMessageHistory]    Script Date: 2017-02-21 09:50:40 ******/
SET ANSI_NULLS ON
GO
SET QUOTED_IDENTIFIER ON
GO
```

**Comment by marco44** [comment]
Should be solved by 4019450e6c9a921a24f4ecdd832647155ce9fe06

**Comment by McDoit** [comment]
Solved!
